site/content/blog/2022-08-30-spier-move/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Spider News"
+summary: >
+  News about changes to the Traditional Spider for the up-coming release.
+images:
+- https://www.zaproxy.org/blog/2022-08-30-spider-move/images/zapbot.spider.jpg
+type: post
+date: "2022-08-30"
+authors:
+    - thorin
+---
+
+# Traditional Spider News
+
+For the upcoming stable release of ZAP a number of changes have been made to the Traditional Spider which are currently available for testing in the weekly releases.
+
+## Moved to an Add-on
+
+To facilitate more frequent functional and bug fixes the core Spider has been moved to an add-on which means such changes are no longer bound to core/stable releases. Other add-ons which use Traditional Spider functionality have also been re-worked to support the Spider add-on, including: QuickStart, FormHandler, GraphQL, OpenAPI, SOAP, and the Automation Framework.
+
+## Functional Improvements Since 2.11
+
+A large number of functional improvements have been made since 2.11. Most of which have been a result of efforts related to [Issue 7152](https://github.com/zaproxy/zaproxy/issues/7152) which is backed by bounties via the Stackhawk [ZAP Fund](https://www.stackhawk.com/zap-fund/).
+
+| Before                                                                                                                                  | After                                                                                                                                                                                                               |
```

Comment:
A description of the table could be added above it, e.g., "The following table lists the changes in the spiders behaviour after the updates."
The headers could be updated to Before 2.12 and After 2.12 (unless you think it's clear from the context).

Reply:
Better not, more changes will be done before 2.12.